testing/trino-tests/src/test/java/io/trino/tests/tpch/TestTpchDistributedStats.java

```diff
@@ -46,7 +47,8 @@ public void setup()
                         // We are not able to calculate stats for PARTIAL aggregations
                         .setSystemProperty(PREFER_PARTIAL_AGGREGATION, "false")
                         // Stats for non-EXPLAIN queries are not collected by default
-                        .setSystemProperty(COLLECT_PLAN_STATISTICS_FOR_ALL_QUERIES, "true"))
+                        .setSystemProperty(COLLECT_PLAN_STATISTICS_FOR_ALL_QUERIES, "true")
+                        .setSystemProperty(DEFAULT_FILTER_FACTOR_ENABLED, "false"))
```

Comment:
ditto could we update tests? Otherwise we might miss behavior changes other CBO changes we make in the future.

Reply:
There isn't much point in updating the tests in this class to assert on guesstimates, I'm going to keep it as is